collectors/cochrane/parser.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from copy import deepcopy
+try:
+    from lxml import etree
+except ImportError:
+    import xml.etree.ElementTree as etree
+from .record import Record
+
+
+def parse_record(url, file_path):
+    tree = etree.parse(file_path)
+    study_robs = []
+    studies = []
+
+    # Get risk of bias
+
+    root = tree.getroot()
+    doi_id = root.attrib.get('DOI', '')
+    quality_item_data_entries = tree.findall('//QUALITY_ITEM_DATA_ENTRY')
+    for quality_item_data_entry in quality_item_data_entries:
+        study_rob = {}
+        study_rob['study_id'] = quality_item_data_entry.attrib['STUDY_ID']
+        study_rob['modified'] = quality_item_data_entry.attrib.get('MODIFIED', '')
+        study_rob['result'] = quality_item_data_entry.attrib['RESULT']
+        study_rob['group_id'] = quality_item_data_entry.attrib.get('GROUP_ID', '')
+        study_rob['group_name'] = ''
+        for description in quality_item_data_entry.iter('P'):
+            study_rob['result_description'] = description.text
```

Comment:
This is overwriting `study_rob['result_description']` with the last `P` element. Is it correct? If so, prefer `quality_item_data_entry.findall('P')[-1].text`, as it looks like a bug for someone without context (like myself :smile:)